### ISSUE TITLE
Reports generator takes echoes for full runs

### ISSUE THREAD

**Issue by oesteban** [poldracklab/fmriprep#1598]

**Comment by jdkent** [comment]
If there are generated figures that have `echo` as a key, then [the refactor](https://github.com/poldracklab/fmriprep/blob/66b39aaca8e2a8c58cda243f3309723db77b8a4d/fmriprep/data/reports-spec.yml#L30) should create a report for those figures. However, the refactor does not generate the figures themselves.

**Comment by oesteban** [comment]
Okay, so before closing this one we just need to check whether the echo- keywords are being propagated through the `DerivativesDataSink`s, correct?

**Comment by jdkent** [comment]
yes, you are correct, I'm unfamiliar with how the multiecho process propagates outputs so I cannot say definitively.
